dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESIndexUtilTest.java

```diff
@@ -0,0 +1,199 @@
+        } finally {
+            putReadOnly(indiciesInfo.getWorking(), false);
+        }
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#putReadOnlyToFalse(String...)}
+     * When: The index not exists
+     * Should: throw a {@link org.elasticsearch.ElasticsearchStatusException}
+     *
+     */
+    @Test(expected = ElasticsearchStatusException.class)
+    public void shouldTryPutReadonlyInTrueAndTheIndexDoesNotExists()  {
+        ESIndexUtil.putReadOnlyToFalse("index_not_exists");
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#isAnyCurrentIndicesReadOnly()}
+     * When: If at least one of the current working index if read only
+     * Should: return true
+     *
+     * @throws DotDataException
+     */
+    @Test()
+    public void shouldReturnTrueWhenWorkingIndexIsReadOnly() throws DotDataException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        try {
+            putReadOnly(indiciesInfo.getLive(), false);
+            putReadOnly(indiciesInfo.getWorking(), true);
+
+            final boolean anyCurrentIndicesReadOnly = ESIndexUtil.isAnyCurrentIndicesReadOnly();
+            assertEquals(true, anyCurrentIndicesReadOnly);
+        } finally {
+            putReadOnly(indiciesInfo.getWorking(), false);
+        }
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#isAnyCurrentIndicesReadOnly()}
+     * When: If at least one of the current live index if read only
+     * Should: return true
+     *
+     * @throws DotDataException
+     */
+    @Test()
+    public void shouldReturnTrueWhenLiveIndexIsReadOnly() throws DotDataException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        try {
+            putReadOnly(indiciesInfo.getLive(), true);
+            putReadOnly(indiciesInfo.getWorking(), false);
+
+            final boolean anyCurrentIndicesReadOnly = ESIndexUtil.isAnyCurrentIndicesReadOnly();
+            assertEquals(true, anyCurrentIndicesReadOnly);
+        } finally {
+            putReadOnly(indiciesInfo.getLive(), false);
+        }
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#putCurrentIndicesToWriteMode()}
+     * When: If at least one of the current live index if read only
+     * Should: return true
+     *
+     * @throws ESResponseException
+     * @throws DotDataException
+     */
+    @Test()
+    public void shouldSetWorkingAndLiveIndexToReadOnly() throws DotDataException, ESResponseException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        try {
+            putReadOnly(indiciesInfo.getLive(), false);
+            putReadOnly(indiciesInfo.getWorking(), false);
+
+            ESIndexUtil.putCurrentIndicesToWriteMode();
+            assertEquals(false, ESIndexUtil.isReadOnly(indiciesInfo.getLive()));
+            assertEquals(false, ESIndexUtil.isReadOnly(indiciesInfo.getWorking()));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use assertTrue(x)/assertFalse(x) instead of assertEquals(true, x)/assertEquals(false, x) or assertEquals(Boolean.TRUE, x)/assertEquals(Boolean.FALSE, x).](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642172)